client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -69,6 +87,7 @@ export function Gutenboard() {
 						/>
 						<Header
 							isEditorSidebarOpened={ isEditorSidebarOpened }
+							next={ goToNextStep }
 							toggleGeneralSidebar={ toggleGeneralSidebar }
 							toggleSidebarShortcut={ toggleSidebarShortcut }
 						/>
```

Comment:
Since we are introducing the second screen, before merging this UI change we should consider adding also the 'back' button to the header for users to be able to navigate between steps (even if we still don't have router in place).

Reply:
Fair point. Do we have mockups for a back button yet? :thinking:

Reply:
(Discussed in Slack, and added per 9795260 and 48a6445)